robolectric/src/main/java/org/robolectric/junit/rules/BackgroundTestRule.java

```diff
@@ -19,20 +19,20 @@
  *
  * <p>Example usage:
  *
- * <pre>{@code
```

Comment:
Here, decided the keep the `<pre>` tag only and escape the `@` using HTML code because if we are keeping the combination of `<pre> {@ code...` then in order to escape the I've tried using `@literal @` but its actually rendering it as it is
```
@literal @
```

Reply:
and If I'm using `@literal @` to escape the `@` symbol in `{@code ..}`, its even giving error while generating the Javadoc

```
"/Applications/Android Studio Preview.app/Contents/jre/Contents/Home/bin/javadoc" -protected -splitindex -d "/Users/sagarkhurana/javadocforrobolectic " @/private/var/folders/sr/ncp1rk8x2wq_wr33rbj432mw0000gn/T/javadoc_args
Loading source file /Users/sagarkhurana/AndroidStudioProjects/robolectric/robolectric/src/main/java/org/robolectric/junit/rules/BackgroundTestRule.java...
Constructing Javadoc information...
Standard Doclet version 11.0.13
Building tree for all the packages and classes...
/Users/sagarkhurana/AndroidStudioProjects/robolectric/robolectric/src/main/java/org/robolectric/junit/rules/BackgroundTestRule.java:22: error: unterminated inline tag
 * <pre>{@code
        ^
/Users/sagarkhurana/AndroidStudioProjects/robolectric/robolectric/src/main/java/org/robolectric/junit/rules/BackgroundTestRule.java:23: error: incorrect use of inline tag
 * @literal @Rule public final BackgroundTestRule backgroundTestRule = new BackgroundTestRule();
   ^
/Users/sagarkhurana/AndroidStudioProjects/robolectric/robolectric/src/main/java/org/robolectric/junit/rules/BackgroundTestRule.java:22: warning: empty <pre> tag
 * <pre>{@code
   ^
/Users/sagarkhurana/AndroidStudioProjects/robolectric/robolectric/src/main/java/org/robolectric/junit/rules/BackgroundTestRule.java:22: error: element not closed: pre
 * <pre>{@code
   ^
Generating /Users/sagarkhurana/javadocforrobolectic /org/robolectric/junit/rules/BackgroundTestRule.html...
Generating /Users/sagarkhurana/javadocforrobolectic /org/robolectric/junit/rules/BackgroundTestRule.BackgroundTest.html...
Generating /Users/sagarkhurana/javadocforrobolectic /org/robolectric/junit/rules/package-summary.html...
Generating /Users/sagarkhurana/javadocforrobolectic /org/robolectric/junit/rules/package-tree.html...
Generating /Users/sagarkhurana/javadocforrobolectic /constant-values.html...
Building index for all the packages and classes...
Generating /Users/sagarkhurana/javadocforrobolectic /overview-tree.html...
Generating /Users/sagarkhurana/javadocforrobolectic /index-files/index-1.html...
Generating /Users/sagarkhurana/javadocforrobolectic /index-files/index-2.html...
Generating /Users/sagarkhurana/javadocforrobolectic /index-files/index-3.html...
Building index for all classes...
Generating /Users/sagarkhurana/javadocforrobolectic /allclasses-index.html...
Generating /Users/sagarkhurana/javadocforrobolectic /allpackages-index.html...
Generating /Users/sagarkhurana/javadocforrobolectic /deprecated-list.html...
Building index for all classes...
Generating /Users/sagarkhurana/javadocforrobolectic /allclasses.html...
Generating /Users/sagarkhurana/javadocforrobolectic /allclasses.html...
Generating /Users/sagarkhurana/javadocforrobolectic /index.html...
Generating /Users/sagarkhurana/javadocforrobolectic /help-doc.html...
3 errors
1 warning

"javadoc" finished with exit code 1
```

Reply:
TLDR: can we keep the `<pre>` tag only